docs/product-updates/2023-08-01-custom-code-samples.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Custom code samples
+tags: [Improvement]
+---
+
+You would like to make your API consumers feel more at home by providing them some in-house custom code sample to use your API?
+
+We now have you covered by supporting the custom specification extension `x-codeSamples`.
+
+Head to [our dedicated help page](/help/doc-code-samples/) on this new specification property to see how to use it and you will be able to make some changes to the content rendered in the right side of your API documentation.
+
+![x-codesamples.gif](/files/help/x-codesamples.gif)
```

Comment:
If suggestion on line 6 is accepted, move this to where the drop down menu is mentionned